.github/pr-title-checker-config.json

```
@@ -4,7 +4,7 @@
 		"color": "B60205"
 	},
 	"CHECKS": {
-		"regexp": "^(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])?|(?:Regression|Chore|Revert|i18n):)|(?:Bump) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Merge master into develop)",
+		"regexp": "^(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](?:[^[]|\\[(ENTERPRISE|APPS)\\])|(?:Regression|Chore|Revert|i18n):)|(?:Bump) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Merge master into develop)",
```

The current regex, my guess, seems to be failing because the first group check is loose - if it is in the given string or not. So as long as the given string prefix *contains* one of the given groups, it's fine. I think the better solution is to introduce another group for just the title prefix, as in 

```suggestion
		"regexp": "^(?:(?:(?:\\[(NEW|BREAK|IMPROVE|FIX)\\](\\[(ENTERPRISE|APPS)\\])?|(?:Regression|Chore|Revert|i18n):)|(?:Bump)) .+|Release [0-9]+\\.[0-9]+\\.[0-9]+|Merge master into develop)",
```

Adding as a suggestion so that the change is more visible

---

very wrong "on behalf of" 🙈

---

Having said that, `[^[]` works too, but is slgihtly less readable without context for those who're not as comfortable with regexes.